### ISSUE TITLE
How to access Tivo on LAN from AWS

### ISSUE THREAD

**Issue by jbaumbach** [natejgreene/alexa_tivo#5]
Let's say the skill is running in AWS as a lambda, how can it contact your Tivo running on your LAN?

**Comment by jradwan** [comment]
It can't, unless you do some port forwarding on your router and expose your TiVo to the internet, which I wouldn't recommend. Theoretically that would work but I haven't attempted it ... with [my fork](https://github.com/jradwan/alexa_tivo_control) of the skill I run the node-js/alexa-app-server locally.

**Comment by jbaumbach** [comment]
Makes sense, just wanted to make sure I wasn't missing something. Thanks!
